source/plugin/ore-publish.rst

```diff
@@ -0,0 +1,56 @@
+======================
+Publishing Your Plugin
+======================
+
+Sponge's official plugin / mod repository, `Ore <http://ore-staging.spongepowered.org>`_, is a free and open-source
+project that anyone may publish their Sponge plugins or Forge mods on.
+
+Packaging Your Plugin
+~~~~~~~~~~~~~~~~~~~~~
+
+Ore requires any projects to be packaged with a ``mcmod.info`` descriptor file in the top-level of your JAR file. This
+file is used to automatically infer some important details about your project to make the upload process easier.
+Luckily, Sponge API has a built in annotation processor that creates this file for you automatically, on compile, using
+the ``@Plugin`` annotation that you have likely already created in your plugin's main class.
```

Comment:
Well the plugin will not run without a `@Plugin` annotation, nor will it without the file, and yes, the upload will be rejected if the file is missing. I'll mention this and add a reference to creating a plugin.